content/release.rst

```diff
@@ -19,6 +19,19 @@ Releasing a product usually consists of publishing the following:
 * Documentation, which includes Release Notes, Known Issues and Upgrade Steps.
 * Public announcement. Notification email or website news/blog article.
 
+Each version sent to a customer should have an unique version number and
+should be part of the release notes history.
+
+When a functionality is released for a certain customer group, but is not
+wanted in the main release series, it should be released from a branch which
+is not removed, a tag is created for each release and all release notes
+are imported into the main branch.
+
+TODO: see what to do with customers using releases from staging... maibe
+make a production release without a full QA review.
+Such a release can have a normal version... and if changes are made
+during the QA/review a new patch version is released.
```

Comment:
May be it would be good to use pre-release version in the same way as it is described in [Semantic Versioning 2.0.0](http://semver.org/):

```
A pre-release version MAY be denoted by appending a hyphen and a series of dot separated identifiers immediately following the patch version. Identifiers MUST comprise only ASCII alphanumerics and hyphen [0-9A-Za-z-]. Identifiers MUST NOT be empty. Numeric identifiers MUST NOT include leading zeroes. Pre-release versions have a lower precedence than the associated normal version. A pre-release version indicates that the version is unstable and might not satisfy the intended compatibility requirements as denoted by its associated normal version. Examples: 1.0.0-alpha, 1.0.0-alpha.1, 1.0.0-0.3.7, 1.0.0-x.7.z.92.
```